python/django/security/path-traversal-join.yaml

```diff
@@ -0,0 +1,45 @@
+rules:
+  - id: path-traversal
+    patterns:
+      - pattern-inside: |
+          def $F(...):
+            ...
+      - pattern-not: |
+          ...
+          os.path.abspath(...)
+          ...
+      - pattern-not: |
+          ...
+          os.path.realpath(...)
+          ...
+      - pattern-either:
+        - pattern: |
+            $V = request.$W.get($X)
+            ...
+            $P = os.path.join(..., $V)
```

Comment:
Sorry! Last nit... would we catch more by doing `(..., $V, ...)` in case they put it in the middle of `os.path.join`?